src/main/java/db/FruitStorage.java

```diff
@@ -0,0 +1,12 @@
+package db;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class FruitStorage {
+    private static Map<String, Integer> fruits = new TreeMap<>();
```

Comment:
For get/put operations HashMap is better choise. TreeMap better if we have to do iterations by keys in sorted order
```suggestion
    private static Map<String, Integer> fruits = new HashMap<>();
```